crypto/err/openssl.ec

```diff
@@ -17,7 +17,7 @@ L ASN1          include/openssl/asn1err.h       crypto/asn1/asn1_err.c
 L CONF          include/openssl/conferr.h       crypto/conf/conf_err.c                  include/crypto/conferr.h
 L CRYPTO        include/openssl/cryptoerr.h     crypto/cpt_err.c                        include/crypto/cryptoerr.h
 L EC            include/openssl/ecerr.h         crypto/ec/ec_err.c                      include/crypto/ecerr.h
-L SSL           include/openssl/sslerr.h        ssl/ssl_err.c                           ssl/sslerr.h
+L SSL           include/openssl/sslerr.h        crypto/ssl_err.c                           crypto/sslerr.h
```

Comment:
Should that not be
```suggestion
L SSL           include/openssl/sslerr.h        crypto/ssl_err.c                           include/crypto/sslerr.h
```
(and of course the `sslerr.h` file has to be moved accordingly).

Reply:
Also, aren't these fooerr.h generated by some ghastly perl script? Doesn't that need adapting?

Reply:
Ah, the perl-generated thing is what is now called `fooerr_legacy.h`. So in the new world order you need one header for a prototype of a silly function that is used in exactly two places. Brilliant.